packages/web-components/src/color-vNext/palette.ts

```diff
@@ -0,0 +1,143 @@
+import { clamp, ColorRGBA64, ComponentStateColorPalette, parseColorHexRGB } from '@microsoft/fast-colors';
```

Comment:
I wonder if general purpose types like `Palette` should actually be in foundation instead?

Reply:
Right now I'm not sure that these won't potentially change or diverge - I do think we can create a corresponding issue in FAST to track potentially breaking some of the color work into it's own package, but I'd prefer a more robust data set before we do that.